app/models/user.rb

```diff
@@ -5,7 +5,11 @@ class User < ApplicationRecord
   rolify
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :authy_authenticatable, :database_authenticatable, :registerable, :validatable, :lockable, :password_expirable
+  if Rails.env.production?
```

Comment:
Our internal test server runs in production, and we don't want 2FA on it. How do we disable it in that case?

Reply:
Perhaps `&& Authy.api_key.present?` or `ENV["AUTHY_API_KEY"].present?`?

Reply:
I've opted for adding it as an admin option in sara.yaml. I did so because I'd much rather have a user have to change the environment variable for 2FA rather than the possible case where they have somehow neglected to include the Authy API key and accidentally disabled all 2FA in the system.

I hope `export SARA_ALERT_TWO_FACTOR_AUTH_ENABLED=false` is an easy fix in the internal test server.